setup-conda/action.yml

```diff
@@ -0,0 +1,39 @@
+name: "Setup conda"
+description: "Install conda Miniconda or Anaconda."
+
+inputs:
+  installer:
+    description: "Version of conda to install"
+    required: true
+    default: "Anaconda"
+  version_anaconda:
```

Comment:
I would prefer to combine these. In the README, we can link to these two URLs, which give an overview of all available versions:
https://repo.anaconda.com/miniconda/
https://repo.anaconda.com/archive/

Reply:
Ah true, I was thinking from the perspective of our usage of the this action. From there I wanted to reduce as much code as possible and just assumed by default everyone wants to install/check directly both Anaconda and Miniconda. This doesn't make sense from the actions perspective. I'll combine it! 
Thank you for your review!!